operators/src/adapters/raster_subquery/raster_subquery_reprojection.rs

```diff
@@ -88,12 +88,17 @@ where
             .and_then(|vo| vo.intersection(&query_rect.spatial_partition()));
         if let Some(bounds) = valid_spatial_bounds {
             let proj = CoordinateProjector::from_known_srs(self.out_srs, self.in_srs)?;
-            let projected_bounds = bounds.reproject(&proj)?;
-            Ok(Some(RasterQueryRectangle {
-                spatial_bounds: projected_bounds,
-                time_interval: TimeInterval::new_instant(start_time)?,
-                spatial_resolution: self.in_spatial_res,
-            }))
+            let projected_bounds = bounds.reproject(&proj);
+
+            match projected_bounds {
+                Ok(pb) => Ok(Some(RasterQueryRectangle {
+                    spatial_bounds: pb,
+                    time_interval: TimeInterval::new_instant(start_time)?,
+                    spatial_resolution: self.in_spatial_res,
+                })),
+                Err(geoengine_datatypes::error::Error::OutputBboxEmpty { bbox: _ }) => Ok(None), // in some strange cases the reprojection can return an empty bbox. YWe ignore it since it contains no pixels.
```

Comment:
```suggestion
                // In some strange cases the reprojection can return an empty box.
                // We ignore it since it contains no pixels.
                Err(geoengine_datatypes::error::Error::OutputBboxEmpty { bbox: _ }) => Ok(None),
```